Pear/Controller/HomeViewController.swift

```diff
@@ -29,14 +29,12 @@ class HomeViewController: UIViewController {
     private let tabs = ["Weekly Pear", "People"]
     private var user: User?
 
-    // Temporary button to trigger in-app match feedback
-    private var feedbackButton = UIButton()
-
     override func viewDidLoad() {
         super.viewDidLoad()
 
         view.backgroundColor = .backgroundLightGreen
         presentAnnouncement(completion: nil)
+        setupLocalNotifications()
```

Comment:
Inconsistent capitalization. On line 83 you have `setUp`. Also, doesn't this still request right when the app first opens? Weren't you going to move this to the end of onboarding for now?

Reply:
this requests when they're in the home screen for the first time which would be right after they first complete onboarding. did you mean to put it at the last screen of onboarding right before they click "ready for pear"? I put it in the home screen because it's a timed notification and if the user doesn't complete onboarding they don't get a pear (I think) so they shouldn't receive a notification even if they allow it.

Reply:
Ok cool that makes sense.